gitcash/src/main.rs

```diff
@@ -33,6 +33,9 @@ enum Command {
 
     /// Interactive CLI
     Cli,
+
+    /// Generate an example config
+    ConnectPointOfSale,
```

Comment:
Can we name this after what it does?

```suggestion
    GenerateConfig,
```

After all, it isn't necessarily the PoS that needs to generate the config. It could be a developer, or some notification script, or something else.